manifests/generated/virt-handler.yaml.in

```diff
@@ -29,6 +29,8 @@ spec:
         - $(NODE_NAME)
         - --pod-ip-address
         - $(MY_POD_IP)
+        - --max-metric-requests
```

Comment:
Looks like you edited this manually. Please don't do that, everything in this `generated` folder is.... generated ;)
You already found the right place to modify this file, `deployments.go`. After doing changes there you need to run `make generate` on order to update this file. That will also make Travis happy, it checks if `make generate` results in file changes.

Reply:
@slintes understood, thanks for the tip! Let me fix this